### ISSUE TITLE
Build is broken in CI

### ISSUE THREAD

**Issue by zaibon** [xmonader/tcprouter#22]

**Comment by zaibon** [comment]
@xmonader why did you close this issue? The problem is not solved.

**Comment by xmonader** [comment]
@zaibon Auto action from github 
![image](https://user-images.githubusercontent.com/64129/74508575-0e82b080-4f08-11ea-87f9-890d23d1a5ec.png)

**Comment by zaibon** [comment]
Yeah exactly. You merged this PR while the test are failing. And this is the reason I open this issue after.
